src/core/CONVENTIONS.md

```diff
@@ -152,10 +152,12 @@ export class MyPlugin implements Plugin {
   public setup(core) {
     core.application.register({
       id: 'my-app',
-      async mount(context, domElement) {
+      async mount(params) {
         // Load application bundle
         const { renderApp } = await import('./application/my_app');
-        return renderApp(context, domElement);
+        // Get start services
+        const [coreStart, depsStart] = core.getStartServices();
+        return renderApp(coreStart, depsStart, domElement);
```

Comment:
renderApp is defined as `renderApp = (core: CoreStart, deps: MyPluginDepsStart, { element, appBasePath }: AppMountParams)`

So this would be `return renderApp(coreStart, depsStart, params);`

Reply:
Ah, yes fixed it in the other location, but missed it here. Thanks!